test/fixtures/poro.rb

```diff
@@ -260,9 +262,4 @@ def maker
   cache only: [:id]
   attributes :id
 end
-
-RaiseErrorSerializer = Class.new(ActiveModel::Serializer) do
-  def json_key
-    raise StandardError, 'Intentional error for rescue_from test'
-  end
-end
+$VERBOSE = verbose
```

Comment:
this file was just full of warnings that aren't really that important and were really hard to fix, mostly due to all the dynamic class generation.